src/libraries/System.Collections/tests/Generic/PriorityQueue/PriorityQueue.Generic.Tests.cs

```diff
@@ -197,6 +197,30 @@ public void PriorityQueue_EnqueueDequeue(int count)
             AssertExtensions.CollectionEqual(expectedItems, queue.UnorderedItems, EqualityComparer<(TElement, TPriority)>.Default);
         }
 
+        [Theory]
+        [InlineData(1)]
```

Comment:
Any reason why `[MemberData(ValidCollectionSizes)]`  can't be used here? If it contains zero, you might be able to work around that by doing
```C#
(TElement Element, TPriority Priority)[] itemsToEnqueue = CreateItems(1 + count * 2).ToArray();
```
when arranging the queue under test.

Reply:
@eiriktsarpalis I consider that but it seemed unnecessary to me since it would require `(1 + count)` in 4 different places.

I could set `count++` as a first step to dry it up and then use `[MemberData(ValidCollectionSizes)]` if that seems like the better option to you.